sdk-java/src/test/java/ly/count/sdk/java/internal/sc_MV_ManuelViewTests.java

```diff
@@ -0,0 +1,237 @@
+package ly.count.sdk.java.internal;
+
+import ly.count.sdk.java.Countly;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class sc_MV_ManuelViewTests {
+    @Before
+    public void beforeTest() {
+        TestUtils.createCleanTestState();
+    }
+
+    @After
+    public void afterTest() {
+        Countly.instance().halt();
+    }
+
+    //(1XX) Value sanitation, wrong usage, simple tests
+
+    /**
+     * recordView(x2), startAutoStoppedView(x2), startView(x2), pauseViewWithID, resumeViewWithID, stopViewWithName(x2),
+     * stopViewWithID(x2), addSegmentationToViewWithID, addSegmentationToViewWithName
+     * <p>
+     * called with "null" values. versions with and without segmentation. nothing should crash, no events should be recorded
+     */
+    @Test
+    public void MV_100_badValues_null() {
+        Countly.instance().init(TestUtils.getConfigViews());
+        TestUtils.validateEQSize(0);
+        Assert.assertEquals(0, TestUtils.getCurrentRQ().length);
+
+        Countly.instance().view(null);
+        Countly.instance().view(null, false);
+        Countly.instance().views().startAutoStoppedView(null);
+        Countly.instance().views().startAutoStoppedView(null, TestUtils.map());
+        Countly.instance().views().startView(null);
+        Countly.instance().views().startView(null, TestUtils.map());
+        Countly.instance().views().pauseViewWithID(null);
+        Countly.instance().views().resumeViewWithID(null);
+        Countly.instance().views().stopViewWithName(null);
+        Countly.instance().views().stopViewWithName(null, TestUtils.map());
+        Countly.instance().views().stopViewWithID(null);
+        Countly.instance().views().stopViewWithID(null, TestUtils.map());
+        Countly.instance().views().addSegmentationToViewWithID(null, TestUtils.map());
+
+        Countly.instance().views().addSegmentationToViewWithName(null, TestUtils.map());
```

Comment:
as our aim is to make these tests global, it would be best to explain why an expected step is omitted.
So for example, if someone looking at MV_100_badValues_null across all SDKs, they should not ponder here if a step was forgotten or the SDK does not have that feature. In this case it is global view segmentation. Either it can be mentioned at the top of this page or commented in specific test cases. What do you think?

Reply:
Adding notes to each test case might be great because each reviewer probably have a look at to the test cases

Reply:
Added note only to the top